src/gwbot/include/gwbot/lib/tg_event.h

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ *  src/gwbot/include/gwbot/lib/tg_event.h
+ *
+ *  Recieve message header for gwbot
+ *
+ *  Copyright (C) 2021  aliwoto
+ */
+
+#ifndef GWBOT__LIB__TG_EVENT_H
+#define GWBOT__LIB__TG_EVENT_H
+
+#include <stdlib.h>
+#include <gwbot/base.h>
+
+/* 
+ * The kind of the message event.
+ */
+typedef enum _msg_kind_t {
+	TEXT_MSG        = 0,
+	PHOTO_MSG       = 1,
+	STICKER_MSG     = 2
+} _msg_kind_t;
```

Comment:
`typedef` is ok, but only the real definition uses underscore at the beginning.

Please change it to
```c
typedef enum _msg_kind_t {
	TEXT_MSG        = 0,
	PHOTO_MSG       = 1,
	STICKER_MSG     = 2
} msg_kind_t;
```